src/i18n/i18n.ts

```diff
@@ -1,22 +1,28 @@
 import i18n from "i18next";
 import { initReactI18next } from "react-i18next";
+import LanguageDetector from 'i18next-browser-languagedetector';
 import enTranslations from "./en.json";
 
-// the translations
-// (tip move them in a JSON file and import them,
-// or even better, manage them via a UI: https://react.i18next.com/guides/multiple-translation-files#manage-your-translations-with-a-management-gui)
 const resources = {
   en: {
     translation: enTranslations,
   },
+  tc: {
+    translation: {
+      "menu": {
+        "logout": "登出"
+      }
+    },
+  }
```

Comment:
create a file for and load it instead (not sure whether 'tc' is a standard language code 🤔 , should probably be zh-tw)